core/src/main/kotlin/org/evomaster/core/search/service/Randomness.kt

```diff
@@ -452,4 +452,10 @@ class Randomness {
 
         return k
     }
+
+    fun randomIPBit() : Int {
+        val k = (0..255).shuffled().first()
```

Comment:
this is still non-deterministic, as `.shuffled()` is not run on the current instance of `random`.
on a collection, you need to call `.shuffle(random)`